app/tests/unit/forms/form/exportService.spec.js

```diff
@@ -269,7 +454,7 @@ describe('_buildCsvHeaders', () => {
     // get result columns if we need to filter out the columns
     const result = await exportService._buildCsvHeaders(form, submissionsExport, 1, fields, true);
 
-    expect(result).toHaveLength(20);
+    expect(result).toHaveLength(29);
```

Comment:
This test was incorrect, so the length is different now that it's returning the proper column names.